### DISCUSSION TITLE
X first release components

### DISCUSSION THREAD

**Discussion by zombieJ** [ant-design/x#34] [Q&A]
> Ref ChatDesign, ProChat, LobeUI

### Basic

- [ ] ChatBox (RFC: https://github.com/ant-design/ant-design/discussions/48374 , https://github.com/ant-design/x/discussions/16)
  - [ ] Typing
  - [ ] Highlight
  - [ ] Preview
  - [ ] Generation
  - [ ] Error
- [ ] ChatInput
  - <img width="1040" alt="截屏2024-07-22 11 01 27" src="https://github.com/user-attachments/assets/9d47cd4e-a6d1-4b0c-9105-9675904cf0a7">
  - [ ] Customize Input editor
  - [ ] File upload
  - [ ] Voice input
- [ ] TokenTag
  - <img width="148" alt="截屏2024-07-22 11 02 27" src="https://github.com/user-attachments/assets/65a3f874-4abc-4b28-9486-da17e8c1f63c">
- [ ] OperationList 
  - <img width="324" alt="截屏2024-07-22 10 46 48" src="https://github.com/user-attachments/assets/08ffe33c-5c96-48cd-9177-8650b8ce7750">
- [ ] GradientButton
  - <img width="215" alt="截屏2024-07-22 10 51 13" src="https://github.com/user-attachments/assets/ae30e5c6-279b-47c3-a2c3-a316b5ee3b3f">
- [ ] MarkdownEditor
- [ ] Swatches
  - <img width="473" alt="截屏2024-07-22 10 55 38" src="https://github.com/user-attachments/assets/ab18dce4-21d2-447c-b7fd-ae90ef478548">
- [ ] GaussianBackground
  - <img width="427" alt="截屏2024-07-22 10 59 29" src="https://github.com/user-attachments/assets/51c7dddc-c725-41de-b6c6-1acc45b58da1">
- [ ] SchemaConfig
  - ![image](https://github.com/user-attachments/assets/584885de-5684-4b1d-a927-b686d68702b6)

### Hooks

- [ ] useChat: Config to connect with AI proxy api for chat content management

### Mobile

- [ ] TabBar
  - <img width="653" alt="截屏2024-07-22 10 56 38" src="https://github.com/user-attachments/assets/eb0ec1b2-603c-4794-b2ec-f983ea4aa934">
- [ ] ChatPanel (Both support mobile or inline PC window)
  - <img width="149" alt="截屏2024-07-22 11 10 23" src="https://github.com/user-attachments/assets/58fc6e58-74ea-42cd-9dcc-5c4828449dbf">

### Maybe in antd

- [ ] CheckCard ([Pro components - CheckCard](https://procomponents.ant.design/en-US/components/check-card))
- [ ] Toolbar (https://github.com/ant-design/ant-design/issues/49126) 
- [ ] Video

### Global

- [ ] AI style theme token or algorithm
  - loading
  - button
  - tag
  - input
  - tabs
  - alert
  - ...
- [ ] ConfigProvider
  - Localization

**Comment by ONLY-yours** [comment]
useChat 这个在 ProChat 里面，并不一定会触发 ai 的 proxy api，大部分都是对现有数据流的处理。

其实好多业务并不想调用 ai，他们只是想让自己看起来，在调用 ai
